src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -104,107 +111,94 @@ const checkRes = check(res, {
 
 </CodeGroup>
 
-### Monitor the load generator server
+### Monitor the load generator machine
 
 If you are running a test for the first time, it's a good idea to keep an eye on the available resources while the test is running.
-The easiest way to do so is to SSH to the server with 3 sessions:
+The easiest way to do so is to start at least 2 terminals one the machine: one to run k6, and others to monitor the CPU, memory and network usage.
+
+We recommend the following command-line tools for monitoring resources:
+
+- CPU and memory: [htop](https://en.wikipedia.org/wiki/Htop) or [nmon](https://nmon.sourceforge.net/).
 
-1. To run k6
-2. To monitor CPU and memory
-3. To monitor the network
+- Network: [iftop](https://en.wikipedia.org/wiki/Iftop) or [nmon](https://nmon.sourceforge.net/).
 
-For monitoring CPU and memory we recommend [htop](https://en.wikipedia.org/wiki/Htop).
-For monitoring network, we recommend [iftop](https://en.wikipedia.org/wiki/Iftop)
+If you prefer graphical applications, you can use your OS's system monitoring tool (e.g. GNOME System
+Monitor or Plasma System Monitor), or use a standalone tool like [SysMonTask](https://github.com/KrispyCamel4u/SysMonTask).
 
 Here's a screenshot of 3 terminal sessions showing k6, iftop and htop.
 ![k6 iftop htop](./images/large-scale-testing-3-terminals.png)
 
+As general guidelines, look for the following:
+- CPU utilization doesn't exceed 90%. If all CPU cores are 100% utilized during the test run, you might notice performance degradation in your test results.
+- Network utilization is at an acceptable level. Depending on your test, you might expect to fully saturate the network bandwidth, or this might be a signal that your test is bound by the available bandwidth. In other scenarios, you might want to minimize network usage to keep costs down.
+- Memory utilization doesn't exceed 90%. If you're close to exhausting available physical RAM, the system might start swapping memory to disk, which will affect performance and system stability. In extreme cases, running out of memory on Linux will cause the system to end the k6 process.
```

Comment:
Should we recommend running without swap? Our loadgens don't use it, AFAIK. cc: @mstoykov

Reply:
I would recommend people to run without swap *in general* ;) The only use at this point IMO are : 
1. systems which don't have enough ram at all
2. hibernation (which is why I still have swap on my laptop)

In (my) practice the difference between rogue process running out of memory and dying and going into swap is just that in the second case it dies a lot slower and later usually thrashing the whole system in the process ;). 

I guess with NVMe SSDs this is a bit better but even on my laptop whenever I go to swap I find out as the whole system goes really slow.

Given that k6 is a performance tool - it performance degrading significantly because swap is used seems like a terrible idea even if the process itself doesn't die.

Reply:
Fully agree, so I think this deserves a separate section that explains it. Will add, thanks.

Reply:
Resolved by 1ed89f2c.